README.md

```diff
@@ -110,3 +110,31 @@ you can try to use the appassembler plugin as described below. The exec:java plu
     # warning this may take quite a bit of time, so you might want to add the -maxPlugins 1 option
     mvn package appassembler:assemble
     sh target/appassembler/bin/app -id com.example.jenkins -www www
+
+Private Update Center
+---------------------
+
+Jenkins offical update center has lots of plugins. You could follow below steps to create your own
+private update center:
+
+1. Create your own certificate. For example:
+
+    openssl genrsa -out rootCA/demo.key 1024
+    openssl req -new -x509 -days 1095 -key rootCA/demo.key \
+        -out rootCA/demo.crt \
+        -subj "/C=CN/ST=GD/L=SZ/O=vihoo/OU=dev/CN=demo.com/emailAddress=demo@demo.com"
+2. Fetch plugins information then generate update.json
+
+    echo "localization-zh-cn=1.580.1" > whiteList.properties
```

Comment:
It is unclear what the property content value is used for, as we only ever check the key exists. This does not appear to select a specific version…?

Reply:
I might consider do it for matching the properties file format. But the value can actually be ignored.